snappy/click.go

```diff
@@ -133,7 +133,7 @@ ubuntu-core-launcher {{.UdevAppName}} {{.AaProfile}} {{.Target}} "$@"
 	var templateOut bytes.Buffer
 	t := template.Must(template.New("wrapper").Parse(wrapperTemplate))
 	wrapperData := struct {
-		AppName     string
+		SnapName    string
 		AppArch     string
 		AppPath     string
```

Comment:
This one is a bit suspect. Is this really the SnapName rather than the AppName?  Is the AppName missing in this case?  For which app inside the snap is the arch/path referring to?